src/op_mode/zone_policy.py

```diff
@@ -13,27 +13,128 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
-
-import argparse
+import typing
 import tabulate
+import sys
+import vyos.opmode
 
 from vyos.config import Config
```

Comment:
It is better to use `ConfigTreeQuery`
```
from vyos.configquery import ConfigTreeQuery
```
And use `get_config_dict`     [def get_config_dict(self, path=[], effective=False, key_mangling=None,](https://github.com/vyos/vyos-1x/blob/873240f6c8783d62f5f7431843f3a689bb1eea1c/python/vyos/configquery.py#L109)

```
config = ConfigTreeQuery()
```

Reply:
Done